docs/product/components/code-blocks.html

```diff
@@ -0,0 +1,419 @@
+---
+layout: page
+title: Code block
+description: Stacks provides styling for code blocks with syntax highlighting provided by highlight.js. Special care was taken to make sure our light and dark themes felt like Stack Overflow while maintaining near AAA color contrasts and still being distinguishable within tritanopia, deuteranopia, and protanopia.
```

Comment:
Suggestion: drop the medical terms and make it so people like myself who had no idea what those terms meant get it without thinking about it.

```suggestion
description: Stacks provides styling for code blocks with syntax highlighting provided by highlight.js. Special care was taken to make sure our light and dark themes felt like Stack Overflow while maintaining near AAA color contrasts and still being distinguishable for those with a color vision deficiency.
```